### PR TITLE
Switch golang to openapi-generator.

### PR DESCRIPTION
**Pull request by brendandburns**
@tomplus Thanks!

### CODE REVIEW

File: openapi/go.sh [hunk 1]
```diff
@@ -43,7 +43,7 @@ pushd "${OUTPUT_DIR}" > /dev/null
 OUTPUT_DIR=`pwd`
 popd > /dev/null
 
-source "${SCRIPT_ROOT}/swagger-codegen/client-generator.sh"
+source "${SCRIPT_ROOT}/openapi-generator/client-generator.sh"
 source "${SETTING_FILE}"
 
 SWAGGER_CODEGEN_COMMIT="${SWAGGER_CODEGEN_COMMIT:-v2.3.0}"; \
```
Comment (tomplus) [inline openapi/go.sh]: This line can be removed or replaced by https://github.com/kubernetes-client/gen/blob/c05fbc861bd7827bd88fd9b910b6eae04382d235/openapi/openapi-generator/client-generator.sh#L44

Comment (brendandburns) [reply openapi/go.sh]: Done.

**Comment by brendandburns** [comment]
@tomplus thanks! comment addressed.

**Pull request merged**